spec/lib/template_processor/template_processor_for_cf_apps_pipeline_spec.rb

```diff
@@ -27,14 +26,14 @@
       elpaaso-sandbox:
         cf_api_url: https://api.my-cloudfoundry.org
         cf_username: my_user
-        cf_password: myPaSsWord
+        cf_password: "'myPaSsWord"
```

Comment:
we can have password sample with "aPwD;%:-)>Ko"

Reply:
It works as well!